### DISCUSSION TITLE
Pesquisa: o perfil da pessoa desenvolvedora fullstack

### DISCUSSION THREAD

**Discussion by woliveiras** [frontendbr/forum#1253] [Discussão]
Fala, galera!

Estou fazendo uma pesquisa visando traçar o perfil de pessoas que trabalham como fullstacks, estou fazendo uma pesquisa questionando sobre seu dia-a-dia, suas características (pessoais e profissionais), formação (ou não), etc.

Essa pesquisa será postada consolidada e também ficará disponível em formato CSV para quem quiser baixar e fazer uma análise mais profunda das respostas.

Respondam, por favor, e me ajudem a divulgar também: [Pesquisa: o perfil da pessoa desenvolvedora fullstack](https://goo.gl/forms/TmUs2YfAlwWObUc13)

**Comment by chicobentojr** [comment]
Respondido.

Tem um erro na palavra seletivo na primeira opção dessa questão, só avisando :v:

![image](https://user-images.githubusercontent.com/9325152/47937324-1a876200-debf-11e8-8231-965099fa338d.png)

**Comment by rodrigooler** [comment]
@woliveiras respondido 👍

**Comment by woliveiras** [comment]
Muito obrigado pelo alerta, @chicobentojr 

Acertei o trem
